### PR TITLE
Added support for Intel ISIS-II filesystem

### PR DESCRIPTION
**Pull request by fulivi**
Hi, this is actually old PR #10281 adapted to changes in 5499683a239.
Thank you.
--F.Ulivi

### CODE REVIEW

File: src/lib/formats/all.cpp [hunk 1]
```diff
@@ -1074,6 +1078,9 @@ void mame_formats_full_list(mame_formats_enumerator &en)
 #ifdef HAS_FORMATS_IMG_DSK
 	en.add(FLOPPY_IMG_FORMAT); // img_dsk.h
 #endif
+#ifdef HAS_FORMATS_FS_ISIS
+		en.add(fs::ISIS);
```
Comment (ajrhacker) [inline src/lib/formats/all.cpp:1082]: Only one tab should be used here.

Comment (fulivi) [reply src/lib/formats/all.cpp]: ok, I fixed the amount of whitespace. When something like this slips through it usually gets fixed by srcclean before release... anything else to fix, please? Something that doesn't involve just ws?

**Comment by fulivi** [comment]
@galibert: is there any real interest in floptool FS drivers? I'm asking you because I understand you are the main author of floptool and various FS drivers and because my Intel ISIS driver has been languishing in two different PRs for almost 2 months now. If you say there's little interest in these drivers, I will close the PR and move on to something else. Thanks.

**Comment by galibert** [comment]
Sorry, I had forgotten about it.  Looks good to me.

**Pull request merged**